src/clj/rems/api/applications.clj

```diff
@@ -191,6 +192,14 @@
       :return Commenters
       (ok (users/get-commenters)))
 
+    (GET "/export/:form-id" []
```

Comment:
if we plan to support other forms of exports, form-id could be a query parameter, so something like `/api/applications/export?form-id=13` instead of `/api/applications/export/13`. Another option would be `/api/applications/export/form/13`.

Reply:
Thanks, that's a good suggestion, especially since it's good to get the API right the first time around.

Decided to use a query parameter instead of defining the form in the URL because if we end up implementing filtering of applications, query parameters can be extended into accepting multiple parameters without "combining styles".